packages/catalog-search/src/SearchHeader.jsx

```diff
@@ -34,6 +34,9 @@ const SearchHeader = ({
     searchQueryFromRefinements = refinements.q;
   }
 
+  // Add filter to exclude video content type
+  const modifiedFilters = filters ? `${filters} AND NOT content_type:"video"` : "NOT content_type:\"video\"";
```

Comment:
I might flip this around and consider only _including_ the content types officially supported by `SearchSuggestions` (courses & programs), rather than excluding unsupported content types.

Rationale being, pathways could also potentially be surfaced, but the `SearchSuggestions` component doesn't support pathways currently. 

By having these filters only include courses & pathways for now prevents similar regressions from occurring if/when new content types are introduced in the future (e.g., where we wouldn't need to make any changes to _exclude_ the new content type).